reference/3.0/Microsoft.PowerShell.Core/About/about_Command_Precedence.md

```diff
@@ -18,7 +18,7 @@ within a session can be hidden or replaced by commands with the same
 name. This article shows you how to run hidden commands and how to avoid
 command-name conflicts.
 
-## Command precedence
+## Command path precedence
```

Comment:
This whole this is command precedence, only a portion of it should be about path